include/libstored/components.h

```diff
@@ -1370,6 +1370,7 @@ namespace stored {
 					m_Ki = 0;
 					m_Kd = 0;
 					m_y_prev = y;
+					m_int = 0;
```

Comment:
Not resetting `m_int` is on purpose. This makes changing the parameters easier and safer, as there won't be any jumps in the output when P/I/D is updated.